src/main/java/org/embulk/output/s3_per_record/S3PerRecordOutputPlugin.java

```diff
@@ -41,6 +43,11 @@
     public static final String KEY_COLUMN_START_MARKER = "${";
     public static final String KEY_COLUMN_END_MARKER = "}";
 
+    private static final Logger logger = Exec.getLogger(S3PerRecordOutputPlugin.class);
+    private static volatile long nextLoggingRowCount = 1000;
+    private static AtomicLong processedRows = new AtomicLong(0);
+    private static long startTime = System.currentTimeMillis();
```

Comment:
If `startTime` is moved, `startTime` is initialized by each thread. and these have different value.
If logger shares progress between threads, `startTime` should be static, shouldn't it?